### PR TITLE
Fix issue #14126 - open_basedir Warning

### PR DESCRIPTION
**Pull request by hazemtorab**
fixing junkins warnings

### CODE REVIEW

**Comment by davecivicrm** [comment]
ok to test

File: CRM/Utils/HttpClient.php [hunk 1]
```diff
@@ -98,7 +98,13 @@ public function fetch($remoteFile, $localFile) {
     }
     curl_setopt($ch, CURLOPT_FILE, $fp);
 
-    curl_exec($ch);
+    if ($this->isRedirectSupported()){
+      curl_exec($ch);
+    }
+    else{
+      $this->curl_exec_follow($ch);
```
Comment (totten) [inline CRM/Utils/HttpClient.php]: Thank you!

How would you feel about removing the call to `curl_exec()` and simply using `curl_exec_follow()` all the time? It seems like the`curl_exec_follow()` would work regardless of whether `open_basedir` is configured, and having one path makes it easier to test/support.

Comment (hazemtorab) [inline CRM/Utils/HttpClient.php]: Yes you are right .. I replaced it and retested and it's working fine in both cases

**Comment by colemanw** [comment]
I think this is supplanted by https://github.com/civicrm/civicrm-core/pull/7186

**Comment by colemanw** [comment]
jenkins, add to whitelist

**Pull request closed without merging**